Finishing up ICoredebug module4 and process11 (#31068)
dotnet/docs: docs/framework/unmanaged-api/debugging/icordebugprocess11-enumerateloaderheapmemoryregions-method.md

---
# ICorDebugProcess11::EnumerateLoaderHeapMemoryRegions Method

> [Supported in .NET Core 5 and later versions.]  

Reviewer: ```suggestion
[Supported in .NET 5 and later versions.]  
```

>  Enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. It is the same information that would be shown running the SOS "eeheap-loader" command.

Reviewer: ```suggestion
 Enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. The information returned is the same that would be shown by using the SOS `eeheap-loader` command.
```

>  Enumerates ranges of native memory that are used by the .NET runtime to store internal data structures that describe .NET types and methods. It is the same information that would be shown running the SOS "eeheap-loader" command. 

Reviewer 2: Previously I remember seeing that comment about handling in-memory updates in three different spots in the documentation you added. Did we get all the places updated?

Author: I looked in the files that I have changed and it looks like all of them say the correct info.

Author: I believe so, I searched through the files I have changed

 **Library:** CorGuids.lib  

>  **.NET Core Versions:** 5.0 and later

Reviewer 3: ```suggestion
 **.NET Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-core-50-md.md)]
```

>  **.NET Versions:** [!INCLUDE[net_current_v452plus](../../../../includes/net-core-50-md.md)]  

Reviewer 3: ```suggestion
 **.NET Versions:** [!INCLUDE[net-core-50-plus](../../../../includes/net-core-50-md.md)]  
```
